integration-tests/src/test/groovy/com/okta/sdk/tests/it/FactorsIT.groovy

```diff
@@ -178,6 +189,43 @@ class FactorsIT extends ITSupport {
         assertThat response.getFactorResult(), is(VerifyUserFactorResponse.FactorResultEnum.SUCCESS)
     }
 
+    @Test
+    void testEmailUserFactor() {
+        User user = randomUser()
+        assertThat user.listFactors(), emptyIterable()
+
+        EmailUserFactor emailUserFactor = client.instantiate(EmailUserFactor)
+            .setFactorType(FactorType.EMAIL)
+            .setProvider(FactorProvider.OKTA)
+            .setProfile(client.instantiate(EmailUserFactorProfile)
+                .setEmail(user.getProfile().getEmail()))
+
+        assertThat emailUserFactor.id, nullValue()
+        // enroll and activate
+        assertThat emailUserFactor, sameInstance(user.enrollFactor(emailUserFactor, false, null, null, true))
+        assertThat emailUserFactor.getStatus(), is(FactorStatus.ACTIVE)
+        assertThat emailUserFactor.id, notNullValue()
+
+        VerifyFactorRequest request = client.instantiate(VerifyFactorRequest)
+        VerifyUserFactorResponse response = emailUserFactor.verify(request, null, null)
+        assertThat response.getFactorResult(), is(VerifyUserFactorResponse.FactorResultEnum.CHALLENGE)
+    }
+
+    @Test
+    void testTokenUserFactorCreation() {
+        User user = randomUser()
+        assertThat user.listFactors(), emptyIterable()
+
+        TokenUserFactor tokenUserFactor = client.instantiate(TokenUserFactor)
+            .setFactorType(FactorType.TOKEN_SOFTWARE_TOTP)
+            .setProvider(FactorProvider.GOOGLE)
```

Comment:
There is a little overlap with the `activateTotpFactor` for this one, maybe changing the name to include GoogleTotp would make it more clear (if that is the goal of this test)

Reply:
Makes sense, yeah that's the idea of this test.

Reply:
renamed test to `testGoogleTotpUserFactorCreation`.